### DISCUSSION TITLE
[Feature] Add GitLab alerting channel

### DISCUSSION THREAD

**Discussion by seanarnold** [grafana/grafana#37775] [Alerting]
GitLab now supports ingesting alerts from various sources. See [alerts](https://docs.gitlab.com/ee/operations/incident_management/alerts.html), and [alerts HTTP endpoints](https://docs.gitlab.com/ee/operations/incident_management/integrations.html). 

To make this integration better it would be nice to have a native integration with GitLab, much like the other [notifiers](https://grafana.com/docs/grafana/latest/alerting/old-alerting/notifications/#list-of-supported-notifiers).

**Comment by seanarnold** [comment]
Are MRs still accepted for the [old alerting](https://grafana.com/docs/grafana/latest/alerting/old-alerting/notifications), or only Grafana 8+ alerts?

**Reply by dnsmichi** [reply]
@seanarnold Since they are called `legacy alerts` but no deprecation, it seems unclear. Can you please help clarify @marefr?

**Reply by marefr** [reply]
> Are MRs still accepted for the [old alerting](https://grafana.com/docs/grafana/latest/alerting/old-alerting/notifications), or only Grafana 8+ alerts?

Currently both needs updates for us to accept PRs but that probably changes when unified alerting being GA/stable expected soon. In this case I closed the PR as you might noticed?

**Comment by dnsmichi** [comment]
PR implementing this feature request: https://github.com/grafana/grafana/pull/38062

**Comment by seanarnold** [comment]
Hi @marefr, thanks for your feedback on the PR. I've now made myself aware of the new [process](https://github.com/grafana/grafana/discussions/36962) for implementing features.

We've since reached out to our community to upvote this discussion if it's something they are interested in, and it seems that this is now the highest voted [discussion](https://github.com/grafana/grafana/discussions) in the Grafana discussion board. 

Given the interest, are you able to promote this discussion to an issue?

**Reply by seanarnold** [reply]
Hey @marefr, hope you had a good weekend. 

Just reaching out to see if you've reviewed this discussion since your last comment. 
Is there anything I can do to help this feature proposal to be accepted?

Cheers 🙂

**Comment by AUtech-dev** [comment]
We also are interested in using this integration. So we have tested the state of https://github.com/grafana/grafana/pull/38062 and added some small fixes to it to enable the automatic alert closing in GitLab. See https://github.com/grafana/grafana/pull/38062#issuecomment-961657943 for details.

In evaluating the importance of this feature please also have a look at the issue at GitLab about this endpoint: https://gitlab.com/gitlab-org/gitlab/-/issues/334351

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
